CHANGELOG.md

```diff
@@ -0,0 +1,624 @@
+## Changelog
+
+### Version 2.64 (2019 Nov 21)
```

Comment:
Could you also add 2.63?

Reply:
The wiki doesn't seem to indicate any changelog. Do you have any idea what that changelog is?

Reply:
Yes, because permissions had been already removed. Basically:
- [JENKINS-59696](https://issues.jenkins-ci.org/browse/JENKINS-59696)
- #197